ale_linters/go/gometalinter.vim

```diff
@@ -6,7 +6,7 @@ if !exists('g:ale_go_gometalinter_options')
 endif
 
 function! ale_linters#go#gometalinter#GetCommand(buffer) abort
-    return 'gometalinter '
+    return 'gometalinter --include=''^' . expand('%:p') . '.*$'' '
```

Comment:
Someone will probably run into shell escaping issues on Windows at some point with some suggestions. Windows is crazy.